doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -159,26 +168,31 @@ Run your PyAEDT script. If the errors still persists, try the following:
     export ANSOFT_DEBUG_MODE=3
 
 
-Turn on the gRPC trace on the server side too:
+Enable the gRPC trace on the server:
 
 .. code:: console
 
     export GRPC_VERBOSITY=DEBUG
     export GRPC_TRACE=all
 
-Then start ansysedt.exe as gRPC server.
+Then start ansysedt.exe as a gRPC server and redirect the output.
 
 .. code:: console
 
-    ansysedt -grpcsrv 50051
+    ansysedt -grpcsrv 50051 > /path/to/file/server.txt
+
+The above command redirects the gRPC trace is
+to the file *server.txt*.
 
-The gRPC trace is printed on the terminal console. Capture the output as the server.txt file.
-In another terminal:
+Open another terminal window to trace the
+gRPC calls on the client where the Python script will be run.
 
 .. code:: console
 
     export GRPC_VERBOSITY=DEBUG
     export GRPC_TRACE=all
 
-Run the PyAEDT script(make sure it is trying to connect to the same port as the gRPC server).
-Capture the output as the client.txt file. Send all the logs generated to Ansys Support.
+Now run the PyAEDT script
+(make sure it connects to the same port as the gRPC server, 50051).
```

Comment:
```suggestion
Now run the PyAEDT script, (making sure it connects to the same port as the gRPC server (50051).
```

Reply:
There will be too many "(" s in the text with this suggestion.